docs/modules/examples/pages/knowledge-graph.adoc

```diff
@@ -0,0 +1,288 @@
+= Knowledge Graph
+
+image::https://colab.research.google.com/assets/colab-badge.svg[align="left",link="https://colab.research.google.com/github/datastax-labs/knowledge-graphs-langchain/blob/main/notebook.ipynb"]
+
+Use RAGStack, https://python.langchain.com/docs/use_cases/graph/constructing/#llm-graph-transformer[`LLMGraphTransformer`], and https://www.datastax.com/products/datastax-astra[DataStax AstraDB] to extract knowledge triples and store them in a vector database.
```

Comment:
It's worth noting there is an alternative way to extract that doesn't use their `LLMGraphTransformer`, and instead uses a `KnowledgeSchema` (something we introduced) that let's you describe the kind of nodes and edges to be extracted.

This test demonstrates it's functionality:

https://github.com/datastax/ragstack-ai/blob/main/libs/knowledge-graph/tests/test_extraction.py

You can define the extractor in code, or via a YAML file:

https://github.com/datastax/ragstack-ai/blob/main/libs/knowledge-graph/tests/marie_curie_schema.yaml

Reply:
Added